apps/extension/src/entrypoints/host.content/style.css

```diff
@@ -50,21 +50,34 @@
   text-decoration: inherit;
 }
 
-.read-frog-translated-content-wrapper[lang='zh'] {
+.read-frog-translated-content-wrapper[lang='zh'],
+.read-frog-translated-content-wrapper[lang='zh'] * {
   font-family:
     -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Microsoft YaHei', 'Heiti SC', 'WenQuanYi Micro Hei',
     'Source Han Sans SC', 'Noto Sans SC', Helvetica, Arial, sans-serif !important;
 }
 
-.read-frog-translated-content-wrapper[lang='zh-TW'] {
+.read-frog-translated-content-wrapper[lang='zh-TW'],
+.read-frog-translated-content-wrapper[lang='zh-TW'] * {
   font-family:
     -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang TC', 'Microsoft JhengHei', 'Heiti TC', 'STHeiti',
     'STXihei', 'Hiragino Sans TC', 'WenQuanYi Zen Hei', 'Noto Sans TC', 'Source Han Sans TC', Helvetica, Arial,
     sans-serif !important;
 }
 
-.read-frog-translated-content-wrapper[lang='ja'] {
+.read-frog-translated-content-wrapper[lang='ja'],
+.read-frog-translated-content-wrapper[lang='ja'] * {
   font-family:
     -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Yu Gothic', 'YuGothic', 'Hiragino Kaku Gothic ProN', 'Meiryo',
     'MS PGothic', 'MS Gothic', 'Noto Sans JP', 'Source Han Sans JP', 'IPAexGothic', Arial, Helvetica, sans-serif !important;
 }
+
+.read-frog-translated-content-wrapper[dir='rtl'],
+.read-frog-translated-content-wrapper[dir='rtl'] * {
+  text-decoration-skip-ink: auto;
+  font-family:
+    'SF Arabic', 'Noto Sans Arabic', 'Geeza Pro', 'Baghdad', 'Segoe UI', Tahoma, Arial, 'Noto Naskh Arabic', sans-serif !important;
+  font-feature-settings:
+    'rlig' 1,
+    'calt' 1;
+}
```

Comment:
I feel these is for Arabic, not for all `rtl`

Reply:
yes, but for now all rtl langs are using Arabic characters